sdk/tables/Azure.Data.Tables/tests/TableClientQueryableLiveTests.cs

```diff
@@ -0,0 +1,477 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.Data.Tables.Queryable;
+using NUnit.Framework;
+
+namespace Azure.Data.Tables.Tests
+{
+    /// <summary>
+    /// The suite of tests for the <see cref="TableServiceClient"/> class.
+    /// </summary>
+    /// <remarks>
+    /// These tests have a dependency on live Azure services and may incur costs for the associated
+    /// Azure subscription.
+    /// </remarks>
+    public class TableClientQueryableLiveTests : TableServiceLiveTestsBase
```

Comment:
I'm concerned that we are adding only a few test for a huge amount of complicated code.

I also think we need add more unit test style tests for query building because a lot of them would be checking if query was generated correctly and do not require service roundtrip.

Reply:
Yeah, I had the same thought. I could create a set of unit tests that only need to call `TableQuery<TElement>.Bind` and then inspect the resulting `FilterString` etc.

Reply:
Now that this has been greatly simplified, I'll defer this test debt to a future PR. Tracking with #12249